config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  root "posts#index"
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :user, only: [:new, :create]
+  resources :sessions, only: [:new, :create, :destroy]
+  get '/signup', to: 'user#new'
```

Comment:
Either use this

```ruby
resources :users, only: [:new, :create] 
```
Or 
```ruby
get '/signup', to: 'user#new'
post '/signup', to: 'user#create'
```
So remove one.

Reply:
Another thing. Again like I said for `posts` controller. User should be Users also in the controller. `UsersController` like this. So the method should be `users#new`. Also, fix this.